src/screens/settings/container/settingsContainer.js

```diff
@@ -74,15 +75,54 @@ class SettingsContainer extends Component {
         break;
 
       case 'api':
-        dispatch(setApi(serverList[action]));
-        setServer(serverList[action]);
+        this._changeApi(action);
         break;
 
       default:
         break;
     }
   };
 
+  _changeApi = async (action) => {
+    const { dispatch, selectedApi } = this.props;
+    const { serverList } = this.state;
+    const server = serverList[action];
+    let serverResp;
+    let isError = false;
+    const client = new Client(server, { timeout: 3000 });
+
+    dispatch(setApi(server));
+
+    try {
+      serverResp = await client.database.getDynamicGlobalProperties();
+    } catch (e) {
+      isError = true;
+      dispatch(toastNotification('Connection Failed!'));
```

Comment:
can you put to locale file

Reply:
Now if I change them app goes crush becouse of language left them for now  then wait for update language pr

Reply:
alright